### PR TITLE
Update ci pod

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Update ci pod:
1. Create separate environment variable for creating ci pod [ci_pod/env_vars_ci_pod.sh](https://github.com/redhat-performance/benchmark-runner/pull/853/files#diff-0ba281ecc4b0d5e12a62dbdf16e71352b9533471394bc5bd0bc925546d18fa02)
2. Adding auto generating CNV workload summary report [i_pod/jupyterlab/generate_summary_report.sh](https://github.com/redhat-performance/benchmark-runner/pull/853/files#diff-85c20012192c3648a0865f39ac9c3dc4c033b491353c6b3e497c6496c5941aee)
3. Update flask listening port
4. create 1 file for run ci pod [ci_pod/run_ci_pod.sh](https://github.com/redhat-performance/benchmark-runner/pull/853/files#diff-869dcc4904d3bd5dc153d0a1557b09baf64862861857a4b187799d88c42485b0)

### CODE REVIEW

**Comment by RobertKrawitz** [comment]
The changes are rather extensive and structural.  Please update the PR description to better explain what it's doing.

File: ci_pod/nginx/nginx.conf [hunk 1]
```diff
@@ -58,3 +58,25 @@ http {
...
+#		protocol   pop3;
+#		proxy      on;
+#	}
+#
+#	server {
+#		listen     localhost:143;
+#		protocol   imap;
+#		proxy      on;
+#	}
+#}
```
Comment (RobertKrawitz) [inline ci_pod/nginx/nginx.conf:82]: Why is this here, especially as it's commented out?

**Review by RobertKrawitz — COMMENTED** [review]
Just wonder why that commented out stanza is added to nginx.conf; otherwise looks good.

File: ci_pod/nginx/nginx.conf (hunk 1 above)
Comment (ebattat) [reply ci_pod/nginx/nginx.conf]: Removed

**Comment by ebattat** [comment]
I think its the default nginx.conf

**Comment by RobertKrawitz** [comment]
> I think its the default nginx.conf

If that change came from the default nginx.conf changing, that's a good reason to make the change -- keep it as close to the default as practicable.

**Review by RobertKrawitz — APPROVED** [review]
